guides/common/modules/con_web-ui-overview.adoc

```diff
@@ -4,7 +4,32 @@
 = {ProjectWebUI} overview
 
 You can manage and monitor your {Project} infrastructure from a browser with the {ProjectWebUI}.
-For example, you can use the following navigation features in the {ProjectWebUI}:
+ifdef::satellite[]
```

Comment:
```suggestion
ifdef::satellite,orcharhino[]
```

I would even argue that it's also fair for the community to only recommend either Chrome or Firefox.

So drop line 7 & 9-24.

Reply:
Dropped!